### ISSUE TITLE
Address attributes missing "country"?

### ISSUE THREAD

**Issue by mleibner** [18F/identity-dev-docs#86]
On https://github.com/18F/identity-dev-docs/blob/master/_pages/attributes.md, the address attributes are listed as `street_address`, `locality` (city), `region` (state), and `postal_code` for IAL2.

The OpenID Connect [address claim](https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim) also lists a `country` field, though, which is not shown in these developer docs.

Was `country` left out of this documentation, or is it not a provided field by the system?

**Comment by zachmargolis** [comment]
Currently we do not return a country field in the address claim

**Comment by mleibner** [comment]
Thanks for the reply. 

Will country be added to the address claim once passport verification is available, to support US passport holders currently living internationally?

**Comment by zachmargolis** [comment]
I don't think we have any current plans --- but I'll check with the team and track this internally

**Comment by porta-antiporta** [comment]
@mleibner This is a request that has come up in other forums. Given the need to validate information against issuing/authoritative sources in order for login.gov to include it in our attribute bundle, `international` is quite a broad scope. We will be looking at the coverage our current service providers outside the US and seek to partner with entities in-country or US entities that have authoritative expat records that can be used to validate user-asserted information.

Realistically, given the long lead time of the latter two, it will be some time before we can support international addresses.

Note, we are treating support for US passports as a strong piece of evidence as a separate feature and that will be available earlier than international addresses.

**Comment by Shahrukh7766** [comment]
Country. Pakistan Karachi 
Transfer my money to direct deposit my easypesa online digital banking account wallet number and national id card thought 
National id card #4220106231843
Easypesa wallet #03113599057
Account name 
Shahrukh husan
